### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kalanza**
I have added a few guidelines on how to contribute to  the project and a code of conduct for TSA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,92 @@
...
+
+Administrators who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+If you are unsure whether an incident is a violation, or whether the space where the incident took place is covered by our Code of Conduct, **we encourage you to still report it**. We would prefer to have a few extra reports where we decide to take no action, than to leave an incident go unnoticed and unresolved that may result in an individual or group to feel like they can no longer participate in the community. Reports deemed as not a violation will also allow us to improve our Code of Conduct and processes surrounding it. If you witness a dangerous situation or someone in distress, we encourage you to report even if you are only an observer.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), [version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
```
Comment (Cli4d) [inline README.md:92]: This should be in a different file. Create a readme file and call it CODEOFCONDUCT.md then place this content in it

**Review by Cli4d — CHANGES_REQUESTED** [review]
Please see my comments to create a file for code of conduct. Otherwise with that change is should be good enough to merge

File: README.md (hunk 1 above)
Comment (Kalanza) [reply README.md:92]: Ok on it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+   - Go to the repository on GitHub.
+   - Click on the “Pull Requests” tab.
+   - Click the “New Pull Request” button.
+   - Select your branch as the base branch (usually main).
+   - Choose the branch with your changes as the compare branch (e.g., feature/my-awesome-feature).
+   - Add a title and description for your PR.
+   - Click “Create Pull Request.”
+
+Note. Remember to replace placeholders like <repository_url> and customize the branch names according to your specific project.
+
```
Comment (Kalanza) [inline README.md:41]: Nice work

**Review by Cli4d — APPROVED** [review]
LGTM ✅

**Pull request merged**